### PR TITLE
Add more hreflangs

### LINKED ISSUE #7201: SEO enhancement: Add hreflang to all Simorgh pages
**Is your feature request related to a problem? Please describe.**
In order to better inform search engines of the language of our content, we should add an hreflang meta element to all Simorgh pages.  NB this implementation differs from that on PAL based on updated SEO guidance.

**Describe the solution you'd like**
The meta tag should resemble:
```
<link rel="alternate" href="https://www.bbc.com/afaanoromoo" hreflang="om">
```

On every page this should use the Canonical URL of that page.

Notes:
* This won't work on Pidgin (a there isn't a recognised language code to use) so there needs to be a mechanism for that service not to have this feature.
* Hreflang is already applied on /news Optimo Articles in a more complicated manner - the current solution there needs to be maintained
* We shouldn't set this on /archive or /scotland either due to multiple domain complications which it's not our problem to solve

The language code for each service should be as follows:

Service | Language code
------------ | -------------
Afaan Oromoo | om
Afrique | fr
Amharic | am
Arabic | ar
Azeri | az
Bengali | bn
Burmese | my
Gahuza | rw
Gujarati | gu
Hausa | ha
Hindi | hi
Igbo | ig
Indonesia | id
Japanese | ja
Korean | ko
Kyrgyz | ky
Marathi | mr
Mundo | es*
Nepali | ne
Pashto | ps
Persian | fa
Pidgin | N/A**
Portuguese | pt*
Punjabi | pa
Russian | ru
Serbian Lat | sr-Latn
Serbian Cyr | sr-Cyrl
Sinhala | si
Somali | so
Swahili | sw
Tamil | ta
Telugu | te
Thai | th
Tigrinya| ti
Turkce | tr
UK China Simp | zh-Hans
UK China Trad | zh-Hant
Ukrainian Ukrainian | uk
Ukrainian Russian | ru
Urdu | ur
Uzbek | uz
Vietnamese | vi
Yoruba | yo
Zhongwen Simp | zh-Hans
Zhongwen Trad | zh-Hant

\* We may decided in future these services should have multiple more specific entries
** There is no 'official' language code for Pidgin, so we can't do this

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Resolves #7201 

**Overall change:**
Adds hreflang SEO elements 

**Code changes:**

- Adds isoLang to a number of service configs where it is relevant and was not already there
- NB comments left in to mirror implementation in existing cases (e.g. Thai) per discussion in previous issue
- Updates tests

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -9933,7 +9933,7 @@
     },
     "eslint-config-prettier": {
       "version": "6.11.0",
-      "resolved": "https://registry.npmjs.org/eslint-config-prettier/-/eslint-config-prettier-6.11.0.tgz",
+      "resolved": "https://npm.morph.int.tools.bbc.co.uk/eslint-config-prettier/-/eslint-config-prettier-6.11.0.tgz",
```
Comment (rhenshaw56) [inline package-lock.json]: This is using the morph registry. not sure this file should change as there's no accompanying dependency updates

**Review by rhenshaw56 — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (jimjohnsonrollings) [reply package-lock.json]: Good spot.  I've annoyingly installed Morph and it keeps doing this.

**Review by rhenshaw56 — APPROVED** [review]

**Review by HarveyPeachey — APPROVED** [review]
LGTM

**Comment by PriyaKR** [comment]
Looks good to me.`hreflangs` is added to all pagetypes canonical and amp except pidgin.

**Pull request merged**